README.md

```diff
@@ -55,11 +55,11 @@ See [loopback-next-example](https://github.com/strongloop/loopback-next-example)
 
 # Documentation
 
-- [API documentation](https://github.com/strongloop/loopback-next/wiki/API-documentation)
+- [API documentation](http://apidocs.loopback.io/)
 - [FAQ](https://github.com/strongloop/loopback-next/wiki/FAQ)
-- [LoopBack 3 vs LoopBack 4](https://github.com/strongloop/loopback-next/wiki/LoopBack-3-vs-LoopBack-4)
-- [Official documentation](https://github.com/strongloop/loopback-next/wiki)
-- [Tutorials and examples](https://github.com/strongloop/loopback-next/wiki/Tutorials-and-examples)
+- [LoopBack 3 vs LoopBack 4](http://loopback.io/doc/en/lb4/Migration-guide.html)
+- [Official documentation](http://loopback.io/doc/en/lb4/)
```

Comment:
I think I would put this at the top of the list.  Seems a bit silly to have it further down after linking out to other pages in the docs.

Reply:
you're right. I'd just remove this then.